lib/if.h

```diff
@@ -290,9 +290,9 @@ struct interface {
 };
 
 RB_HEAD(if_name_head, interface);
-RB_PROTOTYPE(if_name_head, interface, name_entry, if_cmp_func);
+RB_PROTOTYPE(if_name_head, interface, name_entry, if_cmp_func)
```

Comment:
So if RB_PROTOTYPE doesn't need the ';' does RB_HEAD need it then?

Reply:
Yes, RB_HEAD does need it (it defines a struct type)